src/scribe_data/cli/main.py

```diff
@@ -0,0 +1,61 @@
+"""
+Setup and commands for the Scribe-Data command line interface.
+
+.. raw:: html
+    <!--
+    * Copyright (C) 2024 Scribe
+    *
+    * This program is free software: you can redistribute it and/or modify
+    * it under the terms of the GNU General Public License as published by
+    * the Free Software Foundation, either version 3 of the License, or
+    * (at your option) any later version.
+    *
+    * This program is distributed in the hope that it will be useful,
+    * but WITHOUT ANY WARRANTY; without even the implied warranty of
+    * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    * GNU General Public License for more details.
+    *
+    * You should have received a copy of the GNU General Public License
+    * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+    -->
+"""
+
+#!/usr/bin/env python3
+import argparse
+from .list import list_wrapper
+from .query import query_data
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description='Scribe-Data CLI Tool')
+    subparsers = parser.add_subparsers(dest='command', required=True)
+
+    # List command
```

Comment:
I think that a good way to do this would be `# MARK:`, which will further put a section in the minimap. I've been starting to do this in my Python code recently, and it's really great!

Reply:
Check out the new version of [cli/main.py](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/cli/main.py) to see what I mean :) Here's a screenshot of my minimap in VS Code with all the `MARK`s showing:

![Screenshot_2024-06-22 22 06 29](https://github.com/scribe-org/Scribe-Data/assets/24387426/cf0d968c-f621-4fe2-91b3-27e3695e0489)